dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathResult.java

```diff
@@ -93,21 +105,14 @@ public String formatDependencyPaths(Iterable<ClassPathEntry> entries) {
     }
     return message.toString();
   }
-
+  
   /**
-   * Returns mapping from the Maven coordinates to {@link ClassPathEntry}. The keys are the
-   * coordinates of the direct dependencies of the root nodes in {@link #dependencyPaths}. The
-   * values are all {@link ClassPathEntry}s in the subtree of the key.
+   * Returns the classpath entries for the specified coordinates.
+   * 
+   * TODO: Explain why this is plural. Shouldn't there be exactly one?
```

Comment:
As in the previous javadoc, the values are all {@link ClassPathEntry}s in the subtree of the key.

Reply:
Probably not though we can revisit. The focus is on not passing a bunch of weird data structures around. Instead I'd like public classes that represent our entities (ClassFile, ClassPathEntry, ClassPath, Artifact, Dependency, etc.) and operations on those.

Reply:
That's good. Would you revert the javadoc update? The previous javadoc explains why the values are plural.

Reply:
I'm not sure it did. Or rather what it said didn't match the code. Notice that this returns class path entries, **not** dependency paths.

Reply:
How about "Returns the class path entries in the subtree of the specified coordinates"?

Reply:
done